specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json

```diff
@@ -117,7 +117,7 @@
                 "description": "Specifies that the output audio is to be encoded into HE-AAC v2 profile."
               }
             ],
-            "modelAsExtensible": true
+            "modelAsString": true
```

Comment:
Talked to Brian offline about this. This is a breaking change, Brian will talk to dev who proposed this change and get back.

Reply:
Quintin, Anu and Samer agreed that extensible enum specific change will not require a version update.

Reply:
The attribute `modelAsString` has same meaning across language generators. C# used to be the one honoring modelAsExtensible but that is getting deprecated. 

As discussed taking this PR.